Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/deploymentslot/DeploymentSlotPropertyViewPresenter.java

```diff
@@ -50,8 +50,8 @@ protected boolean getPublishingProfile(@NotNull final String subscriptionId, @No
     }
 
     @Override
-    protected WebAppBase getWebApp(@NotNull final String sid, @NotNull final String webAppId,
-                                   @Nullable final String name) throws Exception {
+    protected WebAppBase getWebAppBase(@NotNull final String sid, @NotNull final String webAppId,
+                                       @Nullable final String name) throws Exception {
```

Comment:
remove spaces

Reply:
Those spaces are used to align the parameters.